plugins/inputs/t128_metrics/README.md

```diff
@@ -0,0 +1,34 @@
+# 128T Metrics Input Plugin
+
+The metrics input pluging collects metrics from a 128T instance.
+
+### Configuration
+
+```toml
+# Read metrics from a 128T instance
+[[inputs.t128_metrics]]
+## Required. The base url for metrics collection
+# base_url = "http://localhost:31517/api/v1/router/Fabric128/"
+
+## A socket to use for retrieving metrics - unused by default
+# unix_socket = "/var/run/128technology/web-server.sock"
+
+## The maximum number of requests to be in flight at once
+# max_simultaneous_requests = 20
```

Comment:
Not directly. This is limiting the number of requests in flight to the backend at once. Without this in place, the auth endpoint was getting overwhelmed and rejecting requests. Golang is based on green threads (as opposed to system threads) which run coroutines. The language abstracts away the system threads much like python asyncio, but it has the advantage that it can efficiently use more than one system thread. With the amount of time spent waiting for responses, it is likely all the processing will be handled by a single system thread though the code doesn't directly dictate that.

Reply:
👍